### DISCUSSION TITLE
Original Blender Launcher repository closure

### DISCUSSION THREAD

**Discussion by Lauloque** [Victor-IX/Blender-Launcher-V2#11] [General]
I noticed by trying to open my email notification that Dotbow's Blender Launcher is now archived, and therefore the issue tickets are unavailable, the [Issues URL](https://github.com/DotBow/Blender-Launcher/issues) redirects to the [Pull Requests](https://github.com/DotBow/Blender-Launcher/pulls) (which only the closed ones let us see related old tickets), and clicking any link to a specific issue that isn't in a PR (like [#181](https://github.com/DotBow/Blender-Launcher/issues/181)) seems to lead to a 404 page.

Though at least Dotbow added a link to Victor-IX's repo in his readme  
https://github.com/DotBow/Blender-Launcher

I'm not a Github expert, but is there any way to back up the remaining tickets and discussions here?

**Comment by Aces-and-Jacks** [comment]
Do you happen to know why the original project was archived? Did they maybe not want to maintain it anymore? It might be on the discord I just didn't want to go through joining to check

**Reply by Lauloque** [reply]
[Dotbow put his project on indefinite hold back in October 9th 2022](https://blenderartists.org/t/blender-launcher-standalone-software-client/1230543/237?u=l0lock), I think it was because BL was a project done on his free time and he then got a job that made him unavailable for BL.

**Reply by Aces-and-Jacks** [reply]
Thank you. Well, it's good that someone else is maintaining at least

**Comment by Lauloque** [comment]
I did manage to find Dotbow's closure discussion on WebArchive:  
[Future of devepment · DotBow/Blender-Launcher · Discussion #188 · GitHub](https://web.archive.org/web/20230927014225/https://github.com/DotBow/Blender-Launcher/discussions/188)

Which lets me think we can backup some tickets using that tool as well. Though it would greatly help if we could just have direct access to the originals if they aren't wiped out.
